lib/module.js

```diff
@@ -139,26 +141,35 @@ module.exports = function sentry (moduleOptions) {
         Sentry.captureException(error)
       }))
     })
+  } else {
+    logger.info('Sentry server side errors will not be logged because the disable option has been set')
+    this.addPlugin({
+      src: path.resolve(__dirname, 'sentry.mocked.js'),
+      fileName: 'sentry.mocked.js',
```

Comment:
I think you can skip filename to have it autogenerated.

This way you are also overriding the same file twice, if both server and client are disabled. That can be a problem since code can get transpiled differently for each.

Reply:
Maybe the argument about overwriting the file is not true but anyways there is no need to specify file name.

Reply:
Ah, thanks. I will set it to sentry.server.js and sentry.client.js?

Reply:
> Maybe the argument about overwriting the file is not true but anyways there is no need to specify file name.

I changed it to server and client, as this was already the case for the regular if.